external/dhcpc/dhcpc_lwip.c

```diff
@@ -94,7 +94,6 @@ int dhcp_client_start(const char *intf)
 {
 	int ret = -1;
 	struct req_lwip_data req;
-
```

Comment:
Let's revert unnecessary changes in this file.

Reply:
Yes sure.